package.json

```
@@ -71,12 +73,14 @@
     "test:angular-showcase": "npm run test:e2e --workspace=angular-showcase",
     "test:react-showcase": "npm run test:e2e --workspace=react-showcase",
     "test:vue-showcase": "npm run test:e2e --workspace=vue-showcase",
-    "update:dependency:playwright": "node scripts/update-playwright.js"
+    "update:dependency:playwright": "node scripts/update-playwright.js",
+    "update:fonts": "npm-run-all generate:fonts generate:icon-types"
```

```suggestion
    "update:icon-fonts": "npm-run-all generate:icon-fonts generate:icon-types"
```
What about renaming this for "clarity" ?